projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -1,112 +1,119 @@
 .container {
-  // all css variables used down in product-card
-  --product-card-shadow: rgba(0, 0, 0, 0.08);
-  --product-card-shadow-size: 40px;
-  --product-card-name-font-size: 24px;
-  --product-card-description-font-size: 14px;
-  --product-card-actions-font-size: 12px;
-  --product-card-margin-base: 4px;
-  --product-card-actions-size: 36px;
-  --status-size: 8px;
-  --status-active: #008710;
-  --status-inactive: #646970;
-  --status-error: #B32D2E;
-  --status-plugin_absent: #C3C4C7;
+	// all css variables used down in product-card
+	--product-card-shadow: rgba(0, 0, 0, 0.08);
+	--product-card-shadow-size: 40px;
+	--product-card-name-font-size: 24px;
+	--product-card-description-font-size: 14px;
+	--product-card-actions-font-size: 12px;
+	--product-card-margin-base: 4px;
+	--product-card-actions-size: 36px;
+	--jp-black: #000;
+	--status-size: 8px;
+	--status-active: #008710;
+	--status-inactive: #646970;
+	--status-error: #B32D2E;
+	--status-plugin_absent: #C3C4C7;
+	--jp-underline-thickness: 2px;
 
-  padding: 24px;
-  background: var( --jp-white );
-  border-radius: var( --jp-border-radius );
-  box-shadow: 0 0 var(--product-card-shadow-size) var(--product-card-shadow);
-  height: 100%;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
+	padding: 24px;
+	background: var( --jp-white );
+	border-radius: var( --jp-border-radius );
+	box-shadow: 0 0 var(--product-card-shadow-size) var(--product-card-shadow);
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
 
-  &.is-purchase-required,
-  &.plugin_absent {
-    background: none;
-    box-shadow: none;
-    box-shadow: 0 0 0 1px var( --status-plugin_absent ) inset;
-  }
+	&.is-purchase-required,
+	&.plugin_absent {
+		background: none;
+		box-shadow: none;
+		box-shadow: 0 0 0 1px var( --status-plugin_absent ) inset;
+	}
```

Comment:
This shouldn't be made by our lint? 🤔

Reply:
I think scss files are not being formatted. I hought the same some days ago

Reply:
Yeah, that's something that's still on the to-do list, unfortunately. Related PR: #11767